frontend/src/utils.js

```diff
@@ -71,4 +71,26 @@ export const snakeToCamelCase = (str) =>
 
 const camelToSnakeCase = (str) => str.replace(/([A-Z])/g, "_$1").toLowerCase();
 
+export const storiesToGeoJson = (stories) => {
+  let features = [];
+
+  stories.forEach((story) => {
+    let { latitude, longitude, ...properties } = story;
+
+    features.push({
+      type: "Feature",
+      geometry: {
+        type: "Point",
+        coordinates: [latitude, longitude],
+      },
+      properties: properties,
+    });
+  });
+
+  return {
+    type: "FeatureCollection",
+    features: features,
+  };
+};
+
```

Comment:
@yus252  the code for converting story array to geojson object is here. you might need to modify it a little bit to fit your need...

Reply:
I c. Thanks!